### DISCUSSION TITLE
CRYSTOOLS IMPORT FAILED

### DISCUSSION THREAD

**Discussion by VantaHitam** [crystian/ComfyUI-Crystools#92] [Q&A] [answered]
I having this error, and also at the install missing nodes also occured the import failed, how do i fix, so i can use crystools? i already 'TRY FIX' button, nothing seems to work
![WhatsApp Image 2024-07-31 at 12 00 58 AM](https://github.com/user-attachments/assets/2595b875-c17d-4e7d-8c80-3b273a4e12a9)
![image](https://github.com/user-attachments/assets/e13be00d-15db-4c62-b64b-1e76fe45c3ed)

**Comment by Oxidelic** [comment]
Please, suddenly I have the same problem. On an older installation everything works fine, but today I installed Comfy on 2 different machines and this same problem popped up. I've tried to uninstall/install again via the manager. Tried to use thee fix option. Tried to do  PIP installation... still the same error occurs... Updated everything... still the same error. Please give us some directions what to do.

**Comment by crystian** [comment]
umm, trying....

**Comment by crystian** [comment]
I tried and works, I need more info, please, follow the instruction to create an issue 
https://github.com/crystian/ComfyUI-Crystools/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=

**Reply by VantaHitam** [reply]
Done! thx 4 respond

**Comment by crystian** [comment, marked as answer]
Crystools Updated! Please try to updated (should be 1.16.0)
